api.md

```diff
@@ -1854,13 +1829,26 @@ DROP VIEW <view_name> CASCADE;
 #### Sample Usage [](continuous_aggregate-drop_view-examples)
 Drop existing continuous aggregate.
 ```sql
-DROP VIEW contagg_view CASCADE;
+DROP MATERIALIZED VIEW contagg_view;
 ```
->:WARNING: `CASCADE` will drop those objects that depend on the continuous
-aggregate, such as views that are built on top of the continuous aggregate view.
 
+## refresh_continuous_aggregate() :community_function: [](refresh_continuous_aggregate)
+
+Refresh all buckets of a continuous aggregate inside a window. Note
+that only buckets fully in the window will be refreshed, so if any
+buckets are partially outside the window either at the beginning or
+the end will be excluded from the refresh.
+
+#### Required Arguments [](refresh_continuous_aggregate-required-arguments)
+
+|Name|Description|
+|---|---|
+| `continuous_aggregate` | (REGCLASS) The continuous aggregate to refresh. |
+| `window_start` | (INTERVAL) Start of the window relative to current time to refresh. |
+| `window_end` | (INTERVAL) End of the window relative to current time to refresh. |
```

Comment:
I also think we need to describe somewhere what reasonable values for these parameters are. For instance, window_start should be before window_end. And, schedule_interval should (must) be smaller than window start. Perhaps for a later PR, but I am worried that this will be lost.

Reply:
I have deliberately skipped discussing reasonable values for the parameters here since it is not clear what we should recommend. I think we can handle it in a later PR. I added a note about that to the documentation issue.

Reply:
I am not talking about recommended "defaults", but rather configuration that doesn't make sense. For instance, for the policy to make sense:

* window_start should be larger than window_end
* schedule_interval should be smaller than window_start

and so on.

Reply:
I added a notice about this to the `add_continuous_aggregate_policy` but I do not understand how this affects `refresh_continuous_aggregate`. It might not even be the case that there is a policy defined.